.eslintrc.json

```diff
@@ -25,7 +25,8 @@
       "react/prop-types": [0],
       "react/destructuring-assignment": [0],
       "react/jsx-props-no-spreading": "off",
-      "prettier/prettier": "error"
+      "prettier/prettier": "error",
+      "no-param-reassign": 0
```

Comment:
I don't love the addition of this. Why don't we just use a different name for the variable inside the function rather than reusing a parameter name? Shadowing variables is always a little iffy.